packs/trident-operator-25.10.0/values.yaml

```diff
@@ -0,0 +1,335 @@
+pack:
+  spectrocloud.com/display-name: trident-operator-25.10.0
+  releaseNameOverride:
+      trident-operator-25.10.0: trident-operator
+  namespace: trident
+  content:
+    images:
+      - image: docker.io/netapp/trident-operator:25.10.0
+
+charts:
+  trident-operator-25.10.0:
```

Comment:
the chart name should be `trident-operator` instead of `trident-operator-25.10.0`